src/msg/async/rdma/RDMAServerSocketImpl.cc

```diff
@@ -97,7 +97,7 @@ int RDMAServerSocketImpl::accept(ConnectedSocket *sock, const SocketOptions &opt
     ::close(sd);
     return -errno;
   }
-  net.set_priority(sd, opt.priority);
+  net.set_priority(sd, opt.priority, out->get_family());
```

Comment:
@saritz i'm trying to see why `out` should EVER be null inside `PosixServerSocketImpl::accept`, I'd say we're probably better off with `assert(NULL != out)`